### ISSUE TITLE
RockingBC Linker errors in Visual Studio

### ISSUE THREAD

**Issue by mhscott** [OpenSees/OpenSees#548]
@vavgen,

A few people (@kkolozvari, @ambaker1, and @mhscott) have noticed linker errors when compiling OpenSees.exe in Visual Studio. The linker errors come from RockingBC and it is not clear why because all settings are the same as for other elements.

Do you currently face similar issues?

**Comment by mhscott** [comment]
![image](https://user-images.githubusercontent.com/13142652/109564136-7feebc00-7a95-11eb-805b-e8c148f66956.png)

**Comment by vavgen** [comment]
I'm sorry for the inconvenience. I did face similar issues in the past. I thought the problem was solved when I uploaded the code, because VS did compile successfully on my machine at that moment.
I redownloaded the code now and tried to compile, and the problem came up again. But if I try to edit my .cpp file (some random edit) and build OpenSees again, it compiles successfully! So the problem is not showing up constantly and this is the reason it is hard to pinpoint.
I will look into it and hope to find the solution today.
Thanks again and I'm sorry for the trouble
